src/github.ts

```diff
@@ -85,24 +86,65 @@ export class GitHub {
   apiUrl: string;
   proxyKey?: string;
   request: Function;
+  graphql: Function;
+  probotMode?: boolean;
 
   constructor(options: GitHubOptions) {
     this.token = options.token;
     this.owner = options.owner;
     this.repo = options.repo;
     this.apiUrl = options.apiUrl || 'https://api.github.com';
-    this.octokit = new Octokit({ baseUrl: options.apiUrl });
     this.proxyKey = options.proxyKey;
-    const defaults: { [key: string]: string | object } = {
-      baseUrl: this.apiUrl,
-      headers: {
-        'user-agent': `release-please/${require('../../package.json').version}`,
-        // some proxies do not require the token prefix.
-        Authorization: `${this.proxyKey ? '' : 'token '}${this.token}`,
-      },
-    };
-    //if (options.proxyKey) defaults['key'] = options.proxyKey;
-    this.request = request.defaults(defaults);
+
+    if (options.octokitInstance === undefined) {
+      this.octokit = new Octokit({ baseUrl: options.apiUrl });
+      const defaults: { [key: string]: string | object } = {
+        baseUrl: this.apiUrl,
+        headers: {
+          'user-agent': `release-please/${require('../../package.json').version}`,
+          // some proxies do not require the token prefix.
+          Authorization: `${this.proxyKey ? '' : 'token '}${this.token}`,
+        },
+      };
+      this.request = request.defaults(defaults);
+      this.graphql = graphql;
+    } else {
+      // for the benefit of probot applications, we allow a configured instance
+      // of octokit to be passed in as a parameter.
+      this.probotMode = true;
+      this.octokit = options.octokitInstance;
+      this.request = options.octokitInstance.request;
+      // @ts-ignore // probot adds a graphql parameter to its instance.
+      this.graphql = options.octokitInstance.graphql;
```

Comment:
This feels dirty - can we make `this.graphql` a interface defined by release-please and probot can wrap it's graphql implementation to provide it.